### DISCUSSION TITLE
What does this checkbox do?

### DISCUSSION THREAD

**Discussion by SuperTempAccount** [AutomaApp/automa#1432] [Q&A]
Hey! Anyone know what this checkbox does in the parameters?

I couldn't find information about it in the documentation.

![image](https://github.com/AutomaApp/automa/assets/83566267/741fec51-7312-44d9-8409-7c64b489609c)

Thanks

**Comment by Delusionnnn** [comment]
Turn it on and activate the script. It’s new and allows the parameter prompt to be inside the active tab instead of a popup. Does not work with automa CEB
